stdlib/public/core/ArrayBufferProtocol.swift

```diff
@@ -35,9 +34,9 @@ internal protocol _ArrayBufferProtocol
     initializing target: UnsafeMutablePointer<Element>
   ) -> UnsafeMutablePointer<Element>
 
-  /// Get or set the index'th element.
-  subscript(index: Int) -> Element { get nonmutating set }
-
+  // /// Get or set the index'th element.
+  // subscript(index: Int) -> Element { get nonmutating set }
+  //
```

Comment:
Ah, right. It wanted an `override` for that, because it's redeclaring the one on `Collection`. But I'm not even sure if marking a protocol subscript `nonmutating set` does what was expected here in the first place.